### PR TITLE
Dev plotdata

### PR DESCRIPTION
**Pull request by DumouchelP**
* **Please check if the PR fulfills these requirements**
- [ ] The commit messages follows our guidelines (explicit, in english)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] CHANGELOG.md has been updated

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

* **Other information**:

### CODE REVIEW

File: volmdlr/templates/d3-unpacker.ts [hunk 1]
```diff
@@ -265,7 +265,7 @@ export class D3Unpacker {
             rectangles_data.push(d);
           }
         })
-
+        console.log(paths_data)
         arcs_data.forEach(function(d){
```
Comment (masfaraud) [inline volmdlr/templates/d3-unpacker.ts]: Remove

File: volmdlr/core.py [hunk 2]
```diff
@@ -4429,9 +4431,10 @@ def copy(self):
     def _bounding_box(self):
         bboxes = []
         for primitive in self.primitives:
+            print(primitive)
             if hasattr(primitive, 'bounding_box'):
                 bboxes.append(primitive.bounding_box)
-
+        print(bboxes)
```
Comment (masfaraud) [inline volmdlr/core.py]: remove print

**Comment by DumouchelP** [comment]
MAJ pull request

File: volmdlr/primitives3D.py [hunk 3]
```diff
@@ -451,18 +451,19 @@ def FreeCADExport(self, ip):
...
-        return area
+        # for i in sic[1:]:
+        #     area-=self.contours2D[i].Area()
+        return areas
 
     def Volume(self):
 #        e=extrusion_vector/norm(extrusion_vector)
         z = npy.cross(self.x.vector,self.y.vector)
-        z.Normalize()
+        z = npy.linalg.norm(z)
```
Comment (masfaraud) [inline volmdlr/primitives3D.py]: Use volmdlr methods:
z = x.Cross(y)
z.Normalize()

Comment (DumouchelP) [reply volmdlr/primitives3D.py]: def Volume(self):
        z = self.x.Cross(self.y)
        z.Normalize()
        coeff = npy.dot(self.extrusion_vector, z)
        return self.Area()*coeff

**Pull request merged**